.github/workflows/osx.yml

```diff
@@ -0,0 +1,53 @@
+name: osx
+
+on:
+  push:
+    branches: [dev_wd]
```

Comment:
does this have to be updated to include other private branches, develop and master?

Reply:
see comment linux.yml